Library/Homebrew/diagnostic.rb

```diff
@@ -341,6 +341,19 @@ def check_tmpdir_sticky_bit
       end
       alias generic_check_tmpdir_sticky_bit check_tmpdir_sticky_bit
 
+      def check_xdg_data_dirs
+        xdg_data_dirs = ENV.fetch("HOMEBREW_XDG_DATA_DIRS", nil)
+        return if xdg_data_dirs.blank? || xdg_data_dirs.split("/").include?(HOMEBREW_PREFIX/"share")
```

Comment:
I know this is old code, and I may be confused, but:
```suggestion
        return if xdg_data_dirs.blank? || xdg_data_dirs.split(":").include?(HOMEBREW_PREFIX/"share")
```
?

Reply:
Good catch, @carlocab! I was staring at the code as I moved it over, but couldn't figure out that niggling feeling that something wasn't quite right... 😆